public/content/documentation/web/page-level/navigation-landmark.md

```diff
@@ -82,95 +62,69 @@ This semantic HTML contains all accessibility features by default.
-    <li><a href="/about/">About</a></li>
-    <li><a href="/contact/">Contact</a></li>
-  </ul>
-</div>
-```
-
-## Developer Notes
-
-### Name
-
-- If there are multiple `<nav>` elements (e.g., site menu, pagination, categories), it may be helpful to name them.
-  - Use `aria-label="Menu name"` when there is no visible navigation title.
-  - Use `aria-describedby="menu-name-id"` when the navigation title is a visible heading.
-
-### Role
-
-- Identifies itself as a navigation landmark.
-- Avoid adding ‘menu’ or ‘option’ roles with arrow key event listeners unless building an actual application like Gmail.
-
-### Focus
-
-- When skip links are used, add `tabindex="-1"` so focus can move to the `nav` element, not just bring it into view.
-
-## Additional Criteria
-
-#### Example Navigation
-
-```html
-<a href="#nav-example">Skip to example navigation</a>
-<a href="#">Not the navigation</a>
-<nav tabindex="-1" class="nav-example" id="nav-example">
-  <ul>
-    <li><a href="/">Home</a></li>
-    <li><a href="/about/">About</a></li>
-    <li><a href="/contact/">Contact</a></li>
-    <li><button aria-haspopup="true">Sign in</button></li>
-  </ul>
-</nav>
-```
-
-#### Expanding Navigation Example
+### Keep custom menus as simple as possible
+Use semantic elements where possible.
 
 ```html
 <nav id="example-expanding-nav" class="menu">
   <ul>
     <li>
-      <a class="home" href="/">Home</a>
+      <a class="home" href="/">
+        Home
+      </a>
     </li>
     <li class="expander-group">
-      <button
-        type="button"
-        class="menu expander-toggle"
-        aria-expanded="true"
-        aria-haspopup="true">
+      <button type="button"
+              class="menu expander-toggle"
+              aria-expanded="false"
+              aria-haspopup="true">
         Menu
       </button>
-      <ul class="subnav expander-content" aria-hidden="false">
+
+      <ul class="subnav expander-content">
         <li>
           <a href="/about/">About</a>
-          <button
-            type="button"
-            class="subnav expander-toggle"
-            aria-expanded="false"
-            aria-haspopup="true">
+          <button type="button"
+                  class="subnav expander-toggle"
+                  aria-expanded="false"
+                  aria-haspopup="true">
             <span class="hidden">About</span>
           </button>
-          <ul class="expander-content" aria-hidden="true">
-            <li><a href="/history/">Our history</a></li>
-            <li><a href="/values/">Our values</a></li>
+
+          <ul class="expander-content">
+            <li>
+              <a href="/history/">Our history</a>
+            </li>
+            <li>
+              <a href="/values/">Our values</a>
+            </li>
           </ul>
+
+        </li>
+        <li>
+          <a href="/contact">Contact</a>
         </li>
-        <li><a href="/contact">Contact</a></li>
       </ul>
     </li>
   </ul>
 </nav>
 ```
```

Comment:
The mockup of the above code snippet is not included on the new MagentaA11Y.

Reply:
do you think I should include it / that it's valuable? I wasn't sure